.env

```diff
@@ -1,6 +1,6 @@
 APP_NAME=BALearner
 APP_ENV=local
-APP_KEY=base64:mAhH+QHK5SfI91EAT2M4BtEu0HIQMo2aX8sHXS6HyCE=
+APP_KEY=base64:HxJH/CFzNrInPt/9vuBuCuW6jaXvyJ65mBum3xFou7E=
```

Comment:
This file should be added to the .gitignore file

Reply:
Todo
- Add API authentication 
- Add request validation
- Fix database migration issues